internal/pkg/cli/job_init.go

```diff
@@ -121,6 +175,110 @@ func (o *initJobOpts) Execute() error {
 	return nil
 }
 
+func (o *initJobOpts) askJobType() error {
+	o.JobType = manifest.ScheduledJobType
+	return nil
+}
+
+func (o *initJobOpts) askJobName() error {
+	if o.Name != "" {
+		return nil
+	}
+
+	name, err := o.prompt.Get(
+		fmt.Sprintf(fmtWkldInitNamePrompt, color.Emphasize("name"), color.HighlightUserInput(o.JobType)),
+		fmt.Sprintf(fmtWkldInitNameHelpPrompt, job, o.AppName()),
+		validateSvcName,
+		prompt.WithFinalMessage("Job name:"),
+	)
+	if err != nil {
+		return fmt.Errorf("get job name: %w", err)
+	}
+	o.Name = name
+	return nil
+}
+
+func (o *initJobOpts) askDockerfile() error {
+	if o.DockerfilePath != "" {
+		return nil
+	}
+	df, err := askDockerfile(o.Name, o.fs, o.prompt)
+	if err != nil {
+		return err
+	}
+	o.DockerfilePath = df
+	return nil
+}
+
+func (o *initJobOpts) askSchedule() error {
+	if o.Schedule != "" {
+		return nil
+	}
+	scheduleType, err := o.prompt.SelectOne(
+		jobInitSchedulePrompt,
+		jobInitScheduleHelp,
+		scheduleTypes,
+		prompt.WithFinalMessage("Schedule type:"),
+	)
+	if err != nil {
+		return fmt.Errorf("get schedule type: %w", err)
+	}
+	switch scheduleType {
+	case rate:
+		return o.askRate()
+	case fixedSchedule:
+		return o.askCron()
+	default:
+		return fmt.Errorf("unrecognized schedule type %s", scheduleType)
+	}
+}
+
+func (o *initJobOpts) askRate() error {
+	rateInput, err := o.prompt.Get(
+		jobInitRatePrompt,
+		jobInitRateHelp,
+		validateRate,
+		prompt.WithFinalMessage("Rate:"),
+	)
+	if err != nil {
+		return fmt.Errorf("get schedule rate: %w", err)
+	}
+	o.Schedule = fmt.Sprintf("@every %s", rateInput)
+	return nil
+}
+
+func (o *initJobOpts) askCron() error {
+	cronInput, err := o.prompt.SelectOne(
+		jobInitCronSchedulePrompt,
+		jobInitCronScheduleHelp,
+		presetSchedules,
+		prompt.WithFinalMessage("Fixed Schedule:"),
+	)
+	if err != nil {
+		return fmt.Errorf("get preset schedule: %w", err)
+	}
+	if cronInput != custom {
+		o.Schedule = getPresetSchedule(cronInput)
+		return nil
+	}
+	customSchedule, err := o.prompt.Get(
+		jobInitCronCustomSchedulePrompt,
+		jobInitCronCustomScheduleHelp,
+		validateSchedule,
+		prompt.WithDefaultInput("0 * * * *"),
+		prompt.WithFinalMessage("Custom Schedule:"),
+	)
+	if err != nil {
+		return fmt.Errorf("get custom schedule: %w", err)
+	}
+	o.Schedule = customSchedule
+	return nil
+}
+
+func getPresetSchedule(input string) string {
+	return fmt.Sprintf("@%s", strings.ToLower(input))
+}
```

Comment:
Do we need to worry about some overzealous cron-loving user throwing in their own "@" here? (And their own "@every" above, for rate.) Probably not, right? I feel like your prompt examples are very clear (especially the wording for rate), but just checking.

Reply:
nope! Whatever gets entered here is transcribed verbatim to the manifest and validated with `validateSchedule`.

Reply:
Right, but they'll get errors because of that. I guess that's what we want, though they may be confused.

Reply:
This is a convenience method that should only ever be called when a customer selects something other than "custom" in the the SelectOne prompt immediately before. Maybe it's better to call it `convertPromptOptionToCronDefinitionString()` for clarity.

Reply:
Oh, gotcha. I don't think that change is necessary--that was just me misunderstanding! :)